improver/cli/nbhood_land_and_sea.py

```diff
@@ -62,7 +62,8 @@ def process(cube: cli.inputcube,
             A cube to be processed.
         mask (iris.cube.Cube):
             A cube containing either a mask of topographic zones over land or
-            a land-sea mask.
+            a land-sea mask. If this a land-sea mask land points should be set
```

Comment:
I think this should be `If this is a land-sea mask, land points ...`

Reply:
Done